defaults/main.yml

```diff
@@ -1,3 +1,10 @@
 ---
 scylladb_repo_url: ""
 scylladb_version: ""
+scylladb_cluster_name: ""
```

Comment:
Add documentation to README.md for this.

Reply:
Done